eng/pipelines/templates/jobs/archetype-sdk-nightly.yml

```diff
@@ -105,7 +105,7 @@ jobs:
     dependsOn:
        - 'Build'
 
-    timeoutInMinutes: 300
+    timeoutInMinutes: 180
```

Comment:
I would simply remove the explicit timeout, if it isn't needed to be more then the default of 60 mins then we should just delete this configuration completely.

Reply:
I have discussed with Scott on this one and he did mention that 60 minutes won't be good enough since it is nightly build and waiting for resources. @scbedd